_pages/setup-wallets/paper.md

```diff
@@ -57,9 +57,8 @@ zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo
 #### Save the Output
 1. Write down the whole 24 word phrase (`zoo` repeated 23 times + `buddy` as the 24th word) *offline* on paper only.
 **Do not save this to any computer.**
-1. Scroll down to `Export Public Key Info To Specter Desktop` and click on the `Download` button. 
-This will download a `json` file to your computer, containing the extended public key info.
-Save the file to a DVD-R or USB drive to later share with Specter-Desktop.  
```

Comment:
I think saving along the way is a good idea (even if not strictly required), so you can reverify on the display of each hardware wallet.

Maybe these steps need more detail? I've been waiting on a good 3rd hardware wallet.